### ISSUE TITLE
Баг на странице прогресса перевода

### ISSUE THREAD

**Issue by ReWiG** [LaravelRUS/docs#250]
![изображение](https://user-images.githubusercontent.com/5054805/146184179-e3a343e2-cc51-400b-8b01-de7cc25e1437.png)
Странное поведение сейчас наблюдается на странице https://laravel.su/status/8.x после обновления данных. Коммит один и тот же, но пишет что отстает на 2 коммита. И такая же ситуация у многих других актуальных файлов

**Comment by slider23** [comment]
Сейчас всё изменится.

**Comment by slider23** [comment]
Так, не сейчас, а где-то через полчаса ) Где-то до migration сейчас всё показывается верно, дальше расхождения. Но номера коммитов верные, можно по ним делать дифф.

**Comment by ReWiG** [comment]
> Так, не сейчас, а где-то через полчаса ) Где-то до migration сейчас всё показывается верно, дальше расхождения. Но номера коммитов верные, можно по ним делать дифф.

Похоже опять все сломалось ;)

**Comment by SerafimArts** [comment]
Да, кажется этот кусок надо будет переписать. Уже 2 проблемы в этой части логики.
